Snippets/Testing/Testing_6/UpgradeGuides/6to7/WhenHandling.cs

```diff
@@ -0,0 +1,46 @@
+﻿namespace Testing_6.UpgradeGuides._6to7
```

Comment:
where is this snippet used?

Reply:
@timbussmann it's line 41 of the upgrade guide doc. It shows that the old method created the message for you but the new overload allows you to send an existing message

Reply:
so it shows both versions on the same page?

Reply:
I think so. I'm pretty sure that's what the `AssertSagaCompletionIs` snippet does on the page.

![image](https://user-images.githubusercontent.com/124014/32879885-aa06bb08-cae6-11e7-971b-decea4a91a53.png)